relay-general/derive/src/lib.rs

```diff
@@ -885,12 +885,20 @@ fn parse_field_attributes(
                                         panic!("Got non string literal for trim_whitespace");
                                     }
                                 }
-                            } else if ident == "match_regex" {
+                            } else if ident == "allow_characters" || ident == "deny_characters" {
```

Comment:
Since we have `max_chars` already, let's shorten this to `allow_chars` and `deny_chars`.